Firestore/core/test/unit/local/query_engine_test.cc

```diff
@@ -151,14 +150,15 @@ class IndexFreeQueryEngineTest : public ::testing::Test {
     });
   }
 
-  DocumentSet ExpectIndexFreeQuery(const std::function<DocumentSet(void)>& f) {
-    local_documents_view_.ExpectIndexFreeExecution(true);
+  DocumentSet ExpectOptimizedCollectionScan(
```

Comment:
Give that the next query engine will add indexes, it seems like the old assertion name is actually a better fit for what we're asserting here.